mkt/webpay/tests/test_webpay_jwt.py

```diff
@@ -83,6 +86,22 @@ def test_no_user(self):
         product = urlparse.parse_qs(request['productData'])
         ok_('buyer_email' not in product)
 
+    @mock.patch.object(settings, 'AMO_LANGUAGES', ('en-US', 'es', 'fr'))
+    def test_locales(self):
+        self.addon.name = {
+            'fr': 'Le Vaurien',
+            'es': 'El Mocoso',
+        }
+        self.addon.description = {
+            'fr': u"ceci est une description d'application",
+            'es': u'se trata de una descripción de la aplicación',
+        }
+        self.addon.save()
+
+        token_data = self.decode_token()
+        eq_(token_data['request']['locales'],
+            self.product.localized_properties())
```

Comment:
This test knows a lot about `product.localized_properties()`. I'd rather tests like these just use stubs rather than requiring all of our business logic and django database stuff but there's a lot going on here and that might be too much of a refactor at this point.

I'd rather see something like this instead of setting `addon.name` and `addon.description` which seem irrelevant in this context:

```py
with mock.patch.object(self.product, 'localized_properties') as local_props:
    local_props.return_value = 'foobar'
    token_data = self.decode_token()
    eq_(token_data['request']['locales'], 'foobar')